webpack/assets/javascripts/react_app/components/charts/ChartHeader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+    constructor(props) {
+        super(props);
+        this.bind(
+            'onClick'
+        );
+    }
+
+    onClick() {
+        this.props.onClick();
+    }
+
+    render() {
+        // TODO: refactor to handle tooltip and placement here or in chart box
+        return (
+            <h4 className={this.props.cssClass}
```

Comment:
Unless I'm missing something, the title in the patternfly examples is in the center of the donut (where I do use that class for the top line.) The patternfly example doesn't have a header like we use.